### PR TITLE
76377 Delete should fail when item not voided

### PR DESCRIPTION
**Pull request by janid1967**
Weakness found when coding delete field

### CODE REVIEW

File: src/tests/Equinor.Procosys.Preservation.Infrastructure.Tests/Repositories/ModeRepositoryTests.cs [hunk 1]
```diff
@@ -62,59 +63,68 @@ public async Task GetByIds_KnownId_ReturnsMode()
...
+        public async Task GetById_KnownId_ShouldReturnsMode()
         {
             var result = await _dut.GetByIdAsync(ModeId);
 
             Assert.AreEqual(ModeId, result.Id);
         }
 
         [TestMethod]
-        public async Task GetById_UnknownId_ReturnsNull()
+        public async Task GetById_UnknownId_ShouldReturnsNull()
```
Comment (llykkelarsen) [inline src/tests/Equinor.Procosys.Preservation.Infrastructure.Tests/Repositories/ModeRepositoryTests.cs:98]: ShouldReturn* here and all the above

**Review by llykkelarsen — APPROVED** [review]
w. suggestions

File: src/tests/Equinor.Procosys.Preservation.Infrastructure.Tests/Repositories/ModeRepositoryTests.cs (hunk 1 above)
Comment (janid1967) [reply src/tests/Equinor.Procosys.Preservation.Infrastructure.Tests/Repositories/ModeRepositoryTests.cs:98]: What do you mean?

Comment (llykkelarsen) [reply src/tests/Equinor.Procosys.Preservation.Infrastructure.Tests/Repositories/ModeRepositoryTests.cs:98]: all the methods say shouldreturnS

Comment (janid1967) [reply src/tests/Equinor.Procosys.Preservation.Infrastructure.Tests/Repositories/ModeRepositoryTests.cs:98]: but they do

**Pull request merged**